modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/Godot.SourceGenerators.csproj

```diff
@@ -18,6 +18,7 @@
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <!-- Do not include the generator as a lib dependency -->
     <IncludeBuildOutput>false</IncludeBuildOutput>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
```

Comment:
The warning I was getting is [Scenario 2](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5128#scenario-2). Since this package is a source generator it does not contain anything in `lib/` or `ref/` (only in `analyzers/`) because it's not a library and this warning is complaining that there's no assembly in `lib/` for the TFM `netstandard2.0` that we target.

So I think this situation is solved using this one of the solutions described in the documentation:

> If your package does not contain any `lib/` or `ref/` files and is not a meta-package, it likely does not have any dependencies that the package consumer needs. If you are packing with NuGet's MSBuild Pack target, you can set `<SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>` in any `PropertyGroup` in your project file. If you are using a custom `nuspec` file, remove the `<dependencies>` element.

I don't fully understand though, so maybe I'm wrong.